migrations/v0.18/go.mod

```diff
@@ -0,0 +1,47 @@
+module github.com/go-vela/community/migrations/v0.18
+
+go 1.19
+
+require (
+	github.com/go-vela/server v0.17.0
+	github.com/go-vela/types v0.17.0
```

Comment:
i dont think it matters but should these point to v0.18.0 (when it exists in release form)

Reply:
Yeah it looks like previous migration scripts are a mixed bag with this. Some pull in the latest versions, some pull in the previous versions. I can update it regardless when we cut the actual release